extensions/control-plane/api/management-api/policy-definition-api/src/main/java/org/eclipse/edc/connector/api/management/policy/PolicyDefinitionApiController.java

```diff
@@ -109,6 +111,19 @@ public IdResponseDto createPolicy(PolicyDefinitionRequestDto requestDto) {
 
     }
 
+    @PUT
+    @Path("{policyId}")
+    @Override
+    public void updatePolicy(@PathParam("policyId") String policyId, PolicyDefinitionUpdateDto updateDto) {
```

Comment:
changed it to `updatedPolicyDefinition`, since
```
transformerRegistry.transform(policyDefinition, PolicyDefinition.class)
```
will provide the ambiguous readability

Reply:
@Valid annotation is not missed, it's been **intentionally not used**
@Valid annotation will null/empty check every field and will throw an Exception if all the fields are not provided with the value.  
Since we do not want that restriction as user(s) use-case would need to send empty entry, such as "prohibitions": []